tests/app/conftest.py

```diff
@@ -828,22 +825,21 @@ def letter_volumes_email_template(notify_service):
 
 
 @pytest.fixture
-def notify_service(notify_db, notify_db_session):
-    user = create_user()
+def notify_service(notify_db, notify_db_session, sample_user):
```

Comment:
Only the `sample_user` fixture is needed for this test.

`sample_user` includes `notify_db_session` which in turn includes `notify_db`. Basically, we can remove some fixtures because none of the tests which have been updated to now use `sample_user` need `notify_db_session` or `notify_db`.

Reply:
Nice (grumbles about dependency injection). I've added an extra commit to remove all the redundant DB fixtures.